studio/pages/project/[ref]/reports/database.tsx

```diff
@@ -51,9 +54,12 @@ const DatabaseUsage = observer(() => {
   })
   const databaseSize = data?.result[0].db_size ?? 0
 
-  const databaseSizeUsageRatio = databaseSize / databaseSizeLimit
-  const sizeIsApproaching = databaseSizeUsageRatio >= USAGE_APPROACHING_THRESHOLD
-  const sizeIsExceeded = databaseSizeUsageRatio >= 1
+  const databaseSizeUsageRatio = databaseSize / databaseUsageLimit
```

Comment:
likewise on this, same as comment [here](https://github.com/supabase/supabase/pull/16265/files#r1288604969)